pinot-controller/src/main/java/org/apache/pinot/controller/ControllerLeadershipManager.java

```diff
@@ -35,12 +37,14 @@
   private static final Logger LOGGER = LoggerFactory.getLogger(ControllerLeadershipManager.class);
 
   private HelixManager _helixManager;
+  private ControllerMetrics _controllerMetrics;
   private volatile boolean _amILeader = false;
 
   private Map<String, LeadershipChangeSubscriber> _subscribers = new ConcurrentHashMap<>();
 
-  public ControllerLeadershipManager(HelixManager helixManager) {
+  public ControllerLeadershipManager(HelixManager helixManager, ControllerMetrics controllerMetrics) {
     _helixManager = helixManager;
+    _controllerMetrics = controllerMetrics;
```

Comment:
we should set it to 0 when the constructor is called

Reply:
Done.